ext/standard/array.c

```diff
@@ -6439,7 +6439,7 @@ PHP_FUNCTION(array_reduce)
 	zval args[2];
 	zval *operand;
 	zend_fcall_info fci;
-	zend_fcall_info_cache fci_cache = empty_fcall_info_cache;
```

Comment:
Why is `empty_fcall_info_cache` not just a macro in the first place? A memset should be better than a memcpy. I suppose this will break `zend_fcc_dtor()`.

Reply:
> True but would still not be redundant here 🙂

Indeed. Not objecting to removing those, but we can at least make the required ones cheaper.

Reply:
Ok I can take a look sometime later

Reply:
Btw, I just blurt out what comes to mind during the review, please just ignore things you don't care about. :slightly_smiling_face: I can do it too if you don't care for it.